amoro-format-mixed/amoro-mixed-trino/src/test/java/org/apache/amoro/trino/mixed/TestKeyedTable.java

```diff
@@ -62,7 +62,7 @@ public void testStats() {
         .matches(
             "VALUES "
                 + "('id', NULL, NULL, 0e0, NULL, '1', '4'), "
-                + "('name$name', 429e0, NULL, 0e0, NULL, NULL, NULL), "
+                + "('name$name', 461e0, NULL, 0e0, NULL, NULL, NULL), "
```

Comment:
I have no idea why this change is needed.

Reply:
`data_size` is an estimate and I'm not sure why it changes. I'm guessing it's the encoding strategy that causes the file size to change.